ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -72,6 +72,8 @@ cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
 
 echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_INSTALL_START"
 
+sleep 900
```

Comment:
Looks like this change triggers quite a number of rehearsal jobs?

Reply:
yes, it's for debugging only

Reply:
If you have some alternative way to add it only for OpenStack jobs - it's worth doing it, spawning jobs on AWS is costly.